### PR TITLE
Added Documentation to add connectors avaliable

### PR DESCRIPTION
**Pull request by kishorebhat-p**
*(If this PR fixes a github issue, please add `Fixes #<xyz>`.)*

Fixes #<xyz>

*(or if this PR is one task of a github issue, please add `Master Issue: #<xyz>` to link to the master issue.)*

Master Issue: #<xyz>

### Motivation

* Added README Changes to add connectors supported by Pulsar for ease of understanding *

### Modifications

* READ ME Changes *

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): NO
  - The public API: NO
  - The schema: (yes / no / don't know)NO
  - The default values of configurations: (yes / no)NO
  - The wire protocol: (yes / no)NO
  - The rest endpoints: (yes / no)NO
  - The admin cli options: (yes / no)NO
  - Anything that affects deployment: (yes / no / don't know)NO

### Documentation

  - Does this pull request introduce a new feature? (yes / no)NO
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: distribution/io/src/assemble/README [hunk 1]
```diff
@@ -7,4 +7,43 @@ contains:
  * the connector code plus all the dependencies
 
  * META-INF/DEPEDENCIES file with licensing information for all transitive
-   dependencies
\ No newline at end of file
+   dependencies
+   
+
+The  source/sink connectors are avaliable for below external interfaces
```
Comment (sijie) [inline distribution/io/src/assemble/README]: what is the motivation of this change?

**Comment by tisonkun** [comment]
Closed as stale.

Also, I think that we don't release connectors in this bundled way but actually each one a NAR file.
